src/ui/deleteAccount/ViewDeleteAccount.qml

```diff
@@ -39,30 +71,39 @@ Item {
             when: VPNAuthInApp.state === VPNAuthInApp.StateStart
             PropertyChanges {
                 target: loader
-                source: "ERROR ERROR! TODO"
+                source: "" // TODO: Error! Handle this case.
```

Comment:
Have to check with UX what we would like to do in case we hit a state that indicates an error during the account deletion flow.

Reply:
No needs to do it. This should never happen.

Reply:
OK — in that case I’ll remove the states entirely.